apps/dashboard/src/app/(internal)/event/components/all-attendees-table.tsx

```diff
@@ -10,6 +10,26 @@ import { FilterableTable, arrayOrEqualsFilter } from "src/components/molecules/F
 import { useUpdateAttendeeReservedMutation, useUpdateEventAttendanceMutation } from "../mutations"
 import { openDeleteManualUserAttendModal } from "./manual-delete-user-attend-modal"
 
+const Selections = ({
+  attendance,
+  attendeeSelections,
+}: { attendance: Attendance; attendeeSelections: AttendeeSelectionResponse[] }) => {
```

Comment:
Prefer to declare an interface. Inline object types hurt readability significantly